config/examples/Creality/Ender-3/MKS Robin E3P/Configuration.h

```diff
@@ -1228,7 +1228,7 @@
 #define Z_MIN_POS 0
 #define X_MAX_POS X_BED_SIZE
 #define Y_MAX_POS Y_BED_SIZE
-#define Z_MAX_POS 250
+#define Z_MAX_POS 210
```

Comment:
The Ender-3's max height is 250 (see the other Ender-3 configs), so this shouldn't be changed.